src/main/java/seedu/address/model/order/Deadline.java

```diff
@@ -31,14 +34,23 @@ public Deadline(String deadline) {
         this.deadline = DateTimeUtil.parseDateTime(deadline);
     }
 
-
     /**
      * Returns true if a given string is a valid deadline.
+     * @returns boolean value to indicate true or false on the statement above.
      */
     public static boolean isValidDeadline(String test) {
         return DateTimeUtil.isValidDate(test);
     }
 
+    /**
+     * Returns true if a given Date is after the current time.
+     * @return boolean value to indicate true or false on the statement above.
+     */
+    public static boolean isAfterCurrentDate(Deadline deadlineObj) {
+        LocalDateTime now = LocalDateTime.now();
+        return deadlineObj.deadline.isAfter(now);
```

Comment:
Maybe this should be in `common.DateTimeUtil`
Also not sure if now() returns epoch time which can cause subtle bugs re timezone issues eg malicious user changes timezone?

Reply:
Okay, use ZonedDateTime object instead of LocalDateTime with our application set to ("Asia/Singapore").
Using ZonedDateTime allows us to compare date/time relative the the timezone. This means that even if a user changes his timezone, his timings will be converted to Singapore time and compared with the dates stored, that are also in Singapore time.

Will update in the UG to inform our users this too.